modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/info/item/provider/test/DDMFormValuesInfoFieldValuesProviderTest.java

```diff
@@ -105,6 +122,44 @@ public void testGetInfoFieldValuesCheckboxMultipleDDMFormFieldType()
 				).build()));
 	}
 
+	@Test
+	public void testGetInfoFieldValuesNestedStructure() throws Exception {
+		DDMStructure ddmStructure = _addDDMStructure(
+			_group, "test-structured-content-nested-structure.json");
```

Comment:
Could you rename this file name to `test-ddm-structure-definition-with-nested-fields.json`? 
What do you think?

Reply:
Sure!